nvflare/job_config/fed_job_config.py

```diff
@@ -154,9 +154,10 @@ def _get_server_app(self, config_dir, custom_dir, fed_app):
 
     def _copy_ext_scripts(self, custom_dir, ext_scripts):
         for script in ext_scripts:
-            dest_file = os.path.join(custom_dir, script)
-            module = "".join(script.rsplit(".py", 1)).replace(os.sep, ".")
-            self._copy_source_file(custom_dir, module, script, dest_file)
+            if not os.path.isabs(script):
```

Comment:
What if I want to copy my scripts from an absolute path, e.g. /my_custom_scripts/...? I think this is the wrong place to handle any security concerns. If there's a concern, we should just not allow any custom folder when submitting the job.

Reply:
but we also want to use this job config API to generate a job folder that I can submit. So, in the case there's no security concern, why can I not submit the script using an absolute path?

Reply:
You can still generate the job folder using the job creation API. Only thing is that the absolute path script won't be generated within the job folder. It is assumed to be pre-installed in the particular location. The "submit_job" command can only submit the package within the job folder structure.

Reply:
Changed to the following behavior: (Added the unit tests)

1. When adding an absolute path external script, user also needs to provide the package path to that external script to generate the proper module codes.
2. If the external script exists in the job API running machine, the external script will be packaged into the job config folder.
3. If the external script does not exists in the job API running machine, it will be treated as a pre-installed script on the job running machine. The job creation API will skip the packaging of this script. The job execution will load it from the job execution runtime.